frontend/public/locales/fr/protected-apply-adult-child.json

```
@@ -319,7 +319,7 @@
   "communication-preference": {
     "page-title": "Communication",
     "preferred-method": "Comment aimeriez-vous que la Sun Life communique avec vous?",
-    "preferred-method-help-message": "Si vous êtes admissible, nous communiquerons vos renseignements à la Sun Life du Canada, compagnie d'assurance-vie (Sun Life). La Sun Life sera responsable du traitement de l'adhésion des membres et de l'administration des demandes de réclamations et des prestations du Régime canadien de soins dentaires.",
+    "preferred-method-help-message": "Si vous êtes admissible, nous transmettrons les renseignements vous concernant et concernant votre enfant à la Sun Life. La Sun Life se chargera de traiter la demande d'adhésion des membres et de l’administration des demandes de réclamations du Régime canadien de soins dentaires.",
```

Nitpick (to be consistent with rest of application):

```suggestion
    "preferred-method-help-message": "Si vous êtes admissible, nous transmettrons les renseignements vous concernant et concernant votre enfant à la Sun Life. La Sun Life se chargera de traiter la demande d'adhésion des membres et de l'administration des demandes de réclamations du Régime canadien de soins dentaires.",
```